src/main/java/com/matthewglover/tictactoe/core/SimpleMiniMax.java

```diff
@@ -2,18 +2,18 @@
 
 public class SimpleMiniMax extends MiniMax {
 
-    public static SimpleMiniMax run(Game game) {
-        SimpleMiniMax miniMax = new SimpleMiniMax(game, 0, true);
+    public static SimpleMiniMax run(Game game, int maxSearchDepth) {
```

Comment:
A little weird that this returns a SimpleMiniMax, presumably you query the returned object to get the information out. Perhaps you can just return that information from the call instead of an instance of this class?